types/react-native/index.d.ts

```diff
@@ -23,6 +23,8 @@
 //                 Mike Martin <https://github.com/mcmar>
 //                 Theo Henry de Villeneuve <https://github.com/theohdv>
 //                 Eli White <https://github.com/TheSavior>
+
+
```

Comment:
This is causing CI to fail.

Reply:
removed the white spaces - 933fe8b647b16a9c21b382fa190bc0bbbc182229